.travis.yml

```diff
@@ -1,6 +1,14 @@
 os:
-    - linux
+  - linux
+
+language: python
+
+python:
```

Comment:
@dnadeau4 why do we need this? Does it run the test twice now?

Reply:
I removed python 3.6 since we already do both version in the `conda env`.

Reply:
@doutriaux1 Can you merge it?